site/classes/activecalendarweek.php

```diff
@@ -49,7 +52,7 @@ public function __construct($year=false, $month=false, $day=false, $GMTDiff="non
 	It generates as many rows as set in $numberOfWeeks
 	********************************************************************************
 	*/
-	function showWeeks($numberOfWeeks=1) {
+	public function showWeeks($numberOfWeeks = 1) {
```

Comment:
Class properties must be defined as public, private, or protected. If declared using var or without any declaration, the property will be defined as public.